principles.md

```diff
@@ -0,0 +1,43 @@
+# Principles
+
+The goal of Vac is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+## I. Liberty
+
+We believe in the sovereignty of individuals. As a research organization that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms. This includes being coercion-resistant.
+
+## II. Censorship resistance
+
+We enable free flow of information. No content is under surveillance. We abide by the cryptoeconomic design principle of censorship resistance. Even stronger, we design agnostic infrastructures for information.
+
+## III. Security
+
+We don't compromise on security when building features. We use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+## IV. Privacy
+
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as pseudo-anonymity. Additionally, we strive to provide the right of total anonymity.
+
+## V. Transparency
+
+We strive for complete openness and symmetry of information within the organization, and have no border between our core contributors and our community. We are frank about our shortcomings, especially when making short-term tradeoffs in service of our long-term goals.
+
+## VI. Openness
+
+The software we create is a public good. It is made available via a free and open source license, for anyone to share, modify and benefit from. We believe in permission-less participation.
+
+## VII. Decentralization
+
+We minimize centralization across both the software and the organization itself. In other words, we maximize the number of physical computers composing the network, and maximize the number of individuals who have control over the system(s) we are building.
+
+## VIII. Inclusivity
+
+We believe in fair and widespread access to our software, with an emphasis on ease-of-use. This also extends to social inclusivity, permissionless participation, interoperability, and investing in educational efforts.
+
+## IX. Continuance
+
+We create software incentivized to continue to exist and improve, without the stewardship of a single entity or any of the current team members.
+
+## X. Resourcefulness
+
+We are relentlessly resourceful. As we grow and have ready access to capital, it is our obligation to token holders to fight bureaucracy and inefficiencies within the organization. This means solving problems in the most effective way possible at lower economic costs (in terms of capital, time and resources).
```

Comment:
Token holders might be confusing in the context of Vac.

Reply:
This is true, and something that could warrant adapting the principles a bit more.

However, de facto funding right now comes from Status, so there's room for something here.

One idea would be to say that these were inherited from the Status principles

Reply:
Yes. That links to the above. I automatically assumed, they would be adapted to Vac.
Mentioning they are inherted from Status would make that much clearer :).